spec.yml

```diff
@@ -2029,6 +2038,97 @@ paths:
           schema:
             $ref: '#/definitions/License'
 
+  /teams/:
+    x-resource: Teams
+    get:
+      summary: Get a list of teams
+      tags:
+        - Users
+      parameters:
+        - $ref: '#/parameters/pageSize'
+        - $ref: '#/parameters/page'
+      responses:
+        200:
+          description: SUCCESS
+          schema:
+            $ref: '#/definitions/TeamsPaginated'
+    post:
+      summary: Create a team
+      tags:
+        - Users
+      parameters:
+        - name: team
+          in: body
+          required: true
+          schema:
+            $ref: '#/definitions/Team'
```

Comment:
I think we should create a separate schema for team creation so that fields like `createdAt`, `createdBy` etc. don't show up in the spec as fields that can be set on creation. It would probably reflect the `Team.Create` case class in the API.